### DISCUSSION TITLE
[General] Cannot claim ETH in zksync.io

### DISCUSSION THREAD

**Discussion by alicedu789** [zkSync-Community-Hub/zksync-developers#750] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

we are facing this error when claiming ETH after bridge.

error:
```
entry.MrNZu0eh.js:67 Uncaught (in promise) ContractFunctionExecutionError: The contract function "finalizeWithdrawal" reverted with the following reason:
xx

Contract Call:
  address:   0xd7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb
  function:  finalizeWithdrawal(uint256 _chainId, uint256 _l2BatchNumber, uint256 _l2MessageIndex, uint16 _l2TxNumberInBatch, bytes _message, bytes32[] _merkleProof)
  args:                        (324, 493293, 0, 1233, 0x6c0960f9ad10251af5ece57a63cc4b3e434dde97a7842fe5000000000000000000000000000000000000000000000000002386f26fc10000, ["0xd07f5b330609e53a456206f25a48bcb05545e131058c624281a3e672efc2768d","0xc3d03eebfd83049991ea3d3e358b6712e7aa2e2e63dc2d4b438987cec28ac8d0","0xe3697c7f33c31a9b0f0aeb8542287d0d21e8c4cf82163d0c44c7a98aa11aa111","0x199cc5812543ddceeddd0fc82807646a4899444240db2c0d2f20c3cceb5f51fa","0xe4733f281f18ba3ea8775dd62d2fcd84011c8c938f16ea5790fd29a03bf8db89","0x1798a1fd9c8fbb818c98cff190daa7cc10b6e5ac9716b4a2649f7c2ebcef2272","0x66d7c5983afe44cf15ea8cf565b34c6c31ff0cb4dd744524f7842b942d08770d","0xb04e5ee349086985f74b73971ce9dfe76bbed95c84906c5dffd96504e1e5396c","0xac506ecb5465659b3a927143f6d724f91d8d9c4bdb2463aee111d9aa869874db","0x124b05ec272cecd7538fdafe53b6628d31188ffb6f345139aac3c3c1fd2e470f","0xc3be9cbd19304d84cca3d045e06b8db3acd68c304fc9cd4cbffe6d18036cb13f","0xfef7bd9f889811e59e4076a0174087135f080177302763019adaf531257e3a87","0xa707d1c62d8be699d34cb74804fdd7b4c568b6c1a821066f126c680d4b83e00b","0xf6e093070e0389d2e529d60fadb855fdded54976ec50ac709e3a36ceaa64c291"])
  sender:    0xaD10251af5EcE57a63Cc4B3E434DDe97A7842fe5

Docs: https://viem.sh/docs/contract/estimateContractGas
Version: viem@2.7.16
    at Oh (entry.MrNZu0eh.js:67:4588)
    at Qz (entry.MrNZu0eh.js:71:2412)
    at async WithdrawalSubmitted.vue.DEPP4wnj.js:1:11756
    at async o6 (entry.MrNZu0eh.js:84:177432)
    at async Promise.all (/transaction/index 1)
    at async Ae.me.cache (WithdrawalSubmitted.vue.DEPP4wnj.js:1:11663)
    at async l (entry.MrNZu0eh.js:84:191418)Caused by: ContractFunctionRevertedError: The contract function "finalizeWithdrawal" reverted with the following reason:
xx

Version: viem@2.7.16
    at Oh (entry.MrNZu0eh.js:67:4502)
    at Qz (entry.MrNZu0eh.js:71:2412)
    at async WithdrawalSubmitted.vue.DEPP4wnj.js:1:11756
    at async o6 (entry.MrNZu0eh.js:84:177432)
    at async Promise.all (/transaction/index 1)
    at async Ae.me.cache (WithdrawalSubmitted.vue.DEPP4wnj.js:1:11663)
```
![image](https://github.com/user-attachments/assets/08fa8341-9e84-42ea-b368-4808ac5ef735)
![image](https://github.com/user-attachments/assets/d8505abb-2da1-4953-aa0f-11a385ad9fdd)

**Comment by alhenaai** [comment]
It appears you are encountering an issue with the `finalizeWithdrawal` function when trying to claim ETH after bridging from ZKsync to Ethereum. This error typically indicates a problem with the parameters passed to the contract function or an issue within the contract itself.

Here are a few steps you can take to troubleshoot and potentially resolve this issue:

1. **Verify Contract Parameters**: Ensure that the parameters provided to the `finalizeWithdrawal` function are correct. This includes the chain ID, batch number, message index, transaction number in batch, message, and merkle proof.

2. **Check Contract Status**: Sometimes, the contract might have conditions or states that prevent the execution of certain functions. Verify that the contract is in the correct state to perform the withdrawal.

3. **Consult Documentation**: Review the contract's documentation or any developer guides that might provide insights into why the function is reverting. Sometimes, specific conditions or prerequisites must be met for successful execution.

4. **Contact Support**: If the issue persists, consider reaching out for more direct support. You can discuss this further with the ZKsync developers on their [GitHub discussions page](https://github.com/zkSync-Community-Hub/zksync-developers/discussions) for more in-depth technical assistance.

If you need further assistance, please provide additional details such as the transaction hash or specific error messages beyond "xx" to help diagnose the problem more accurately.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by alicedu789** [comment]
Hi team,
we would need zysync team to fix it because we are seeing the error in
official site. thanks

entry.MrNZu0eh.js:67 Uncaught (in promise) ContractFunctionExecutionError:
The contract function "finalizeWithdrawal" reverted with the following
reason: xx Contract Call: address:
0xd7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb function:
finalizeWithdrawal(uint256 _chainId, uint256 _l2BatchNumber, uint256
_l2MessageIndex, uint16 _l2TxNumberInBatch, bytes _message, bytes32[]
_merkleProof) args: (324, 493293, 0, 1233,
0x6c0960f9ad10251af5ece57a63cc4b3e434dde97a7842fe5000000000000000000000000000000000000000000000000002386f26fc10000,
["0xd07f5b330609e53a456206f25a48bcb05545e131058c624281a3e672efc2768d","0xc3d03eebfd83049991ea3d3e358b6712e7aa2e2e63dc2d4b438987cec28ac8d0","0xe3697c7f33c31a9b0f0aeb8542287d0d21e8c4cf82163d0c44c7a98aa11aa111","0x199cc5812543ddceeddd0fc82807646a4899444240db2c0d2f20c3cceb5f51fa","0xe4733f281f18ba3ea8775dd62d2fcd84011c8c938f16ea5790fd29a03bf8db89","0x1798a1fd9c8fbb818c98cff190daa7cc10b6e5ac9716b4a2649f7c2ebcef2272","0x66d7c5983afe44cf15ea8cf565b34c6c31ff0cb4dd744524f7842b942d08770d","0xb04e5ee349086985f74b73971ce9dfe76bbed95c84906c5dffd96504e1e5396c","0xac506ecb5465659b3a927143f6d724f91d8d9c4bdb2463aee111d9aa869874db","0x124b05ec272cecd7538fdafe53b6628d31188ffb6f345139aac3c3c1fd2e470f","0xc3be9cbd19304d84cca3d045e06b8db3acd68c304fc9cd4cbffe6d18036cb13f","0xfef7bd9f889811e59e4076a0174087135f080177302763019adaf531257e3a87","0xa707d1c62d8be699d34cb74804fdd7b4c568b6c1a821066f126c680d4b83e00b","0xf6e093070e0389d2e529d60fadb855fdded54976ec50ac709e3a36ceaa64c291"])
sender: 0xaD10251af5EcE57a63Cc4B3E434DDe97A7842fe5 Docs:
https://viem.sh/docs/contract/estimateContractGas Version: ***@***.*** at
Oh (entry.MrNZu0eh.js:67:4588) at Qz (entry.MrNZu0eh.js:71:2412) at async
WithdrawalSubmitted.vue.DEPP4wnj.js:1:11756 at async o6
(entry.MrNZu0eh.js:84:177432) at async Promise.all (/transaction/index 1)
at async Ae.me.cache (WithdrawalSubmitted.vue.DEPP4wnj.js:1:11663) at async
l (entry.MrNZu0eh.js:84:191418)Caused by: ContractFunctionRevertedError:
The contract function "finalizeWithdrawal" reverted with the following
reason: xx Version: ***@***.*** at Oh (entry.MrNZu0eh.js:67:4502) at Qz
(entry.MrNZu0eh.js:71:2412) at async
WithdrawalSubmitted.vue.DEPP4wnj.js:1:11756 at async o6
(entry.MrNZu0eh.js:84:177432) at async Promise.all (/transaction/index 1)
at async Ae.me.cache (WithdrawalSubmitted.vue.DEPP4wnj.js:1:11663)

Regards,

Alice Du

**Reply by bxpana** [reply, marked as answer]
Thanks for reaching out, this is a known issue and something that the team is looking into. For now if you run into issues on the portal you can try claiming on one of the other portals in the ecosystem
- https://syncswap.xyz/move
- https://app.txsync.io/

You can follow the status of the issue for https://portal.zksync.io/bridge here: https://github.com/matter-labs/dapp-portal/issues/193

**Comment by tydcrypt** [comment]
Hello ZkSync team, I'm having issue withdrawing my funds on zkSync bridge. "Fee estimation error: Address is not available" is the error displayed. Please, I'd like the team help to resolve that. Thanks
<img width="1366" height="768" alt="Screenshot (2)" src="https://github.com/user-attachments/assets/0c57526e-23f9-4960-a477-13b00e1da949" />
